sys/posix/include/semaphore.h

```diff
@@ -34,6 +34,9 @@
 extern "C" {
 #endif
 
+/**
+ * @brief A sema_t is a sem_t is a sema_t
```

Comment:
To be honest, I don't even understand the reasoning behind this typedef. Anyhow it does pretty exactly what the doc-line says.